docker-compose.dev.yml

```
@@ -28,9 +28,12 @@ services:
       npm run start:dev -- --preserveWatchOutput"
   db:
     container_name: sos-rs-db
-    image: postgres
+    image: postgres:13.15-alpine3.19
```

```suggestion
    image: postgres:13-alpine
```
Não existe necessidade de fixar o segundo dígito na versão do Postgres pois ela índica uma minor release que contém apenas correções de bug/segurança (https://www.postgresql.org/support/versioning/) então o dump será 100% compatível. Fixando o segundo dígito da versão corre-se o risco de executar postgres com bugs e/ou falha de segurança.

E ao menos que existe uma razão muito específica não vejo porque também fixar a versão do alpine.

---

Outro ponto a levar em consideração, porque versão 13 se a 16 é a última stable?